index.njk

```diff
@@ -5,32 +5,27 @@ title: Rehydrate
 
 <section class="middle">
   <div class="podcast-summary">
-    <p>A podcast where we read books together at a TV series-like pace.</p>
-    <p>Books include <i>The Three-Body Problem</i> and <i>The Long Way to a Small, Angry Planet</i></p>
+    <p>A podcast where we discuss books, music, and other media</p>
   </div>
 
   <section id="how-to-listen">
     <div class="listening-directions">
-      Point your podcast app at the <a href="rss.xml">RSS feed</a> to listen!
+        <a href="rss.xml">RSS</a>,
+        <a href="https://podcasts.apple.com/us/podcast/rehydrate/id1540884633">Apple Podcasts</a>, 
+        <a href="https://pca.st/5w38zj1a">Pocket Casts</a>
```

Comment:
I think this needs to be a list. It's hard to tell the links apart.
![Screenshot_2021-03-10 Rehydrate](https://user-images.githubusercontent.com/324298/110730949-6b29cb00-81ef-11eb-91cc-31f77484a835.png)

Reply:
How's this?

![image](https://user-images.githubusercontent.com/26289/110733908-a3bfa980-81db-11eb-925d-14bc190ca7ce.png)

Reply:
That formatting works, but I think phrasing like "RSS (use this most podcast apps)" will help people with an app that we don't know.

Reply:
I was trying to make that box smaller.. but added it

![image](https://user-images.githubusercontent.com/26289/110889180-8bff2880-82a2-11eb-9d58-3e8cc02327e0.png)

Reply:
Heh, you can drop Overcast to make space. But that's fine as it is with me.